src/ApiService/ApiService/OneFuzzTypes/Enums.cs

```diff
@@ -294,6 +294,7 @@ public enum AgentMode {
 }
 
 public enum NodeState {
+    New,
```

Comment:
Need to add new State for the node (this bug is also in Python code) See "Explanation" for why

Reply:
We convert enums to strings, so order of the Enum members does not matter